index.html

```diff
@@ -125,15 +128,14 @@
     // define bikeTrailsOptions function
     const bikeTrailsOptions =
       function(feature) {
+
         // shortcut to variable
         let type = feature.properties.path_type;
+
         // assign options
-        return type === "Proposed" ?
-          proposedOptions :
-          type === "Shared Roadway (Bikes and Automobiles)" ?
-          sharedOptions :
-          type === "Bikes and Pedestrians Only" ?
-          notSharedOptions :
+        return type === 'Proposed' ? proposedOptions :
+          type === 'Shared Roadway (Bikes and Automobiles)' ? sharedOptions :
+          type === 'Bikes and Pedestrians Only' ? notSharedOptions :
```

Comment:
conditional (ternary) operators were written differently by Rich.  I've been in the habit of using Atom to "beautify" code, and when I did so, it reformatted.  And I see that you also formatted the operators differently from both Rich and the Atom 'beautifier'.  I dont see alot of code in the wild....is your method commonly used in your pro map office?